neurodsp/spectral/utils.py

```diff
@@ -46,8 +46,9 @@ def trim_spectrum(freqs, power_spectra, f_range):
 
     # Restrict freqs & psd to requested range. The if/else is to cover both 1d or 2d arrays
     freqs_ext = freqs[f_mask]
+
     power_spectra_ext = power_spectra[f_mask] if power_spectra.ndim == 1 \
-        else power_spectra[:, f_mask]
+        else power_spectra[f_mask, :]
```

Comment:
Thank you for the detailed comment. Basically the reason I changed some of the slicing and transposing is because `scipy.signal.spectrogram` outputs a 2D array where the horizontal slices are signals across time, and the vertical slices are spectra. I made this commit shortly after noticing `compute_wavelet_transform` had this *flipped*, so I thought it was an error.

Here is an example that I was imagining would be the typical use case.

```python
from neurodsp.utils.data import create_times
from neurodsp.plts import plot_time_series
from neurodsp.sim import sim_oscillation
from neurodsp.spectral import trim_spectrum
from scipy.signal import spectrogram
import matplotlib.pyplot as plt
import numpy as np


n_seconds = 5
fs = 100
times = create_times(n_seconds, fs)

# Create a sinusoid of frequency 1.
sig = sim_oscillation(n_seconds, fs, 1, n_cycles=5)

freqs, t_bins, spg = spectrogram(sig, fs, nperseg=100)
fmask = range(10)

freqs_ext, spg_ext = trim_spectrum(freqs, spg, [0,10])

# Only look at the first 10 frequencies.
plt.pcolormesh(t_bins, freqs_ext, spg_ext, shading='gouraud')
```